bokehjs/src/lib/embed/index.ts

```diff
@@ -15,6 +15,21 @@ export {add_document_from_session} from "./server"
 export {embed_items_notebook, kernels} from "./notebook"
 export {BOKEH_ROOT, inject_css, inject_raw_css} from "./dom"
 
+type JsonItem = {doc: DocJson, root_id: string, target_id: string}
+interface Roots {[index: string]: string}
+
+export function embed_item(item: JsonItem) {
```

Comment:
All types of exported members must be exported. Also, explicit return type is needed. Also, you could use either `type` or `interface` consistently. It (almost) doesn't matter which one is used (until we get into inheritance, which isn't in the picture her).

Reply:
will update